src/Components/ZipCodeForm.tsx

```diff
@@ -10,36 +12,37 @@ const ZipCodeForm = () => {
   const [mileage, setMileage] = useState<ZipCodeSearch['mileage'] | 0 | null>(null)
   let navigate = useNavigate()
 
-  const getOpportunities = (e:React.MouseEvent) => {
+  const getOpportunities = (e: React.MouseEvent) => {
```

Comment:
Per Chris's suggestion, we should deconstruct mouse and click events in the React import to remove React from type